### PR TITLE
Add meme script

### PR DESCRIPTION
**Pull request by ShikherVerma**

### CODE REVIEW

File: scripts/natkhat-meme.coffee [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+#   <Quote natkhat statement from Adarsh> natkhat adarsh - Shows the natkhat adarsh meme
+#
+# Notes:
+#   None
+#
+# Author:
+#   shikher
+
+module.exports = (robot) ->
+  robot.hear /(.*) natkhat adarsh/i, (msg) ->
```
Comment (adarshaj) [inline scripts/natkhat-meme.coffee]: There's a typo here, its just 'natkhat baccha'

File: scripts/natkhat-meme.coffee [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    generateMeme msg, 6913117, 16473882, msg.match[1], "Natkhat Adarsh"
```
Comment (adarshaj) [inline scripts/natkhat-meme.coffee]: It should be, "Natkhat Shikher"

**Comment by ShikherVerma** [comment]
It's already correct. @vikraman plz come to rescue and merge this pr. Adarsh won't admit this pr is correct

**Comment by adarshaj** [comment]
@ShikherVerma

**Pull request merged**